airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -588,21 +599,27 @@ def fetch_requested_container_logs(
             pod_logging_statuses.append(status)
         return pod_logging_statuses
 
+    @tenacity.retry(
+        retry=tenacity.retry_if_exception_type(ResourceNotReadyException),
+        wait=tenacity.wait_exponential(max=15),
```

Comment:
i think there is `wait_random_exponential` which, who knows, could be better

Reply:
Yeah, that's a thing.  I'm not sure how much that would really change things, but I could definitely use that instead if you think it's worthwhile.